quarkus-build-caching-extension/pom.xml

```diff
@@ -143,6 +143,15 @@
                 </executions>
             </plugin>
         </plugins>
+        <resources>
```

Comment:
You'll notice in the CCUD Maven PR I redefined `src/main/resources` as explained [in this comment](https://github.com/gradle/common-custom-user-data-maven-extension/pull/241#discussion_r1743820327).

I think it would be nice to do the same here too for consistency, unless you or Etienne don't agree with my reasoning.

Reply:
Sure thing